src/applications/accredited-representative-portal/accreditation/21a/config/constants.js

```diff
@@ -0,0 +1 @@
+export const transformedData = {};
```

Comment:
Rather than mutate a constant, can we design this as `output = transform(input)`?

Reply:
Sounds good.